client/packages/common/src/intl/locales/en/replenishment.json

```diff
@@ -60,7 +60,7 @@
   "label.requested-packs": "Requested packs",
   "label.quantities": "Quantities",
   "label.amc": "AMC",
-  "label.cant-delete-disabled": "Can only delete lines when status is New",
+  "label.inbound-shipment-cant-delete-reserved-line": "Batch (item code {{itemCode}}) is already reserved/issued",
```

Comment:
Maybe display the batch as well if it's available?

Reply:
Good idea:

![Uploading image.png…]()